README.md

```diff
@@ -146,27 +163,8 @@ jobs:
 The "Delete Staging Deployment" action allows you to delete the deployment not receiving production traffic. This frees up resources used by that deployment and makes room for a new staging deployment:
 
 ```yml
-name: AzureSpringCloud
-on: push
-env:
-  ASC_PACKAGE_PATH: ${{ github.workspace }}
-  AZURE_SUBSCRIPTION: <azure subscription name>
-
-jobs:
-  delete_staging_deployment:
-    runs-on: ubuntu-latest
-    name: Delete staging deployment
-    steps:
-      - name: Checkout Github Action
-        uses: actions/checkout@master
-
-      - name: Login via Azure CLI
-        uses: azure/login@v1
-        with:
-          creds: ${{ secrets.AZURE_CREDENTIALS }}
-
       - name: Delete staging deployment step
-        uses: azure/spring-cloud-git-action@v1
+        uses: azure/spring-cloud-deploy@v1
         with:
           azure-subscription: ${{ env.AZURE_SUBSCRIPTION }}
           action: Delete Staging Deployment
```

Comment:
It looks wired that our action names have space delimiters. Is this from the practice in Azure Pipeline Tasks implementation?